frontend/src/components/card/PlayerCard.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { UserNicknameText } from '../core/Text';
+import { User } from '../../api/User';
+
+const Content = styled.div`
+  display: inline-block;
+  position: relative;
+  margin: 10px;
+  padding: 10px;
+  background-color: ${({ theme }) => theme.colors.lightBlue};
+  border-radius: 5px;
+  
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+type PlayerCardProps = {
+  user: User,
+  isHost: boolean,
+  children: React.ReactNode,
+};
+
+function PlayerCard(props: PlayerCardProps) {
+  const { user, isHost, children: actionCard } = props;
+
+  const [showActionCard, setShowActionCard] = useState(false);
+
+  const toggleActionCard = () => {
+    // Only toggle showActionCard if this user is the host
+    if (actionCard) {
+      setShowActionCard(!showActionCard);
+    }
+  };
+
+  return (
+    <Content onClick={toggleActionCard}>
```

Comment:
Could we do this onhover of the card, instead of onclick?

Reply:
Nice suggestion! I implemented this, although it required a slight CSS gimmick involving an invisible border in order to get it to work (there's a gap between the player name and action card, so without some invisible content, moving the cursor up towards the action card would cause it to leave the component hide the action card before you can use it). If that looks bad, lmk and I can undo it.

Reply:
Yeah I figured that's why this would be difficult, and possible why you didn't implement it at first -- good solution though, I like it! I think it looks great 👍